config/locales/fr.yml

```diff
@@ -516,9 +516,9 @@ fr:
     already_ordered:
       cart: "panier"
       message_html: "Vous avez déjà passé une commande pour ce cycle de vente. Vérifiez votre %{cart} pour voir les produits commandés. Vous pouvez annuler ou modifier votre commande jusqu'à la fermeture du cycle de vente."
-  home:
+  shops:
```

Comment:
This is a good example where the Code Climate engines could come handy, the [grep](https://docs.codeclimate.com/docs/grep) one for instance.